package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "typescript-wallet-sdk",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "license": "MIT",
```

```suggestion
  "license": "Apache-2.0",
```
We can grab the [license file](https://github.com/stellar/kotlin-wallet-sdk/blob/main/LICENSE) from Kotlin Wallet SDK.

---

https://github.com/stellar/typescript-wallet-sdk/pull/1/commits/40929fbefa3d3784c50a43e8f25d79cfa4b86661